### ISSUE TITLE
Best Hyperparameters for version 0.9.0

### ISSUE THREAD

**Issue by OrysB** [AASHISHAG/deepspeech-german#31]
Dear @AASHISHAG ,

thanks a lot for this awesome repository. I am currently trying to export the version 0.9.0 model to the openVINO toolkit.

So far, with the help of the openVINO support, I was able to convert the tensorflow model to the optimized format. It's running with a demo of the open_vino_model_zoo, but the results are far behind to those I get with the mozilla deepspeech example.
A part of the problem I think are the alpha and beta parameters for the language model. How do I find the best values? So far I have seen the best results by setting both to 0. There are values in the flags.txt file in your google drive, but using those there isn't any output at all. 
I don't know if this is of any interest to you or this repository, but I would be very thankful if you could help me out or point me in the right direction.

Greetings
Orys

**Comment by AASHISHAG** [comment]
Hi, @OrysB , I haven't used **openVINO** toolkit before, therefore, I won't be able to comment on this side.

However, regarding LM parameters, I happen to use the default parameters. But these parameters are language-dependent, and therefore, won't work for every use case. You can do a quick grid-search and find the best parameters by decoding on a small dataset.

**Comment by OrysB** [comment]
@AASHISHAG Thanks for the quick reply!

So it turns out that model runs just fine with openVINO and default parameters, if I only use the CPU and no additional hardware from Intel (NCS2), which is why I am doing this in the first place. I found similar issues and it seems to be a bug. Right now I am building the most recent release of the openVINO toolkit from source, hoping it already got fixed. Anyway nothing that concerns this repository. ^^

Thanks again and happy new year!
Orys
